src/main/java/cat/udl/eps/entsoftarch/webingogeiapi/domain/GameStatus.java

```diff
@@ -0,0 +1,9 @@
+package cat.udl.eps.entsoftarch.webingogeiapi.domain;
```

Comment:
This file already exists in master. Merge the last version of master before the Pull Request